projects/experimental/directives/surface/surface.style.less

```diff
@@ -164,3 +164,20 @@ input[tuiSurfaceLayer] {
         }
     }
 }
+
+[tuiSurface][data-surface='dark'] {
+    background: linear-gradient(90deg, rgba(125, 140, 160, 1) 0%, rgba(100, 115, 130, 1) 31%);
+
+    button&,
+    a& {
```

Comment:
Maybe we need to abstract this behavior into a mixin, if it is going to repeat itself for surface presets. Like `.surface-scale()`

Reply:
another PR